docs/user_guide/assessment.rst

```diff
@@ -652,28 +557,23 @@ mentioned above (:meth:`MetricFrame.group_min`, :meth:`MetricFrame.group_max`,
 This takes an underlying metric function, the name of the desired transformation, and
 optionally a list of parameter names which should be treated as sample aligned parameters
 (such as `sample_weight`).
-Other parameters will be passed to the underlying metric function normally (unlike
-:class:`MetricFrame` where :func:`functools.partial` must be used, as noted above).
 The result is a function which builds the :class:`MetricFrame` internally and performs
 the requested aggregation. For example:
 
 .. doctest:: assessment_metrics
     :options:  +NORMALIZE_WHITESPACE
 
     >>> from fairlearn.metrics import make_derived_metric
-    >>> fbeta_difference = make_derived_metric(metric=skm.fbeta_score,
+    >>> recall_difference = make_derived_metric(metric=recall_score,
     ...                                        transform='difference')
-    >>> # Don't need functools.partial for make_derived_metric
-    >>> fbeta_difference(y_true, y_pred, beta=0.7,
-    ...                  sensitive_features=group_membership_data)
-    0.752525...
-    >>> # But as noted above, functools.partial is needed for MetricFrame
-    >>> fbeta_07 = functools.partial(skm.fbeta_score, beta=0.7)
-    >>> MetricFrame(metrics=fbeta_07,
+    >>> recall_difference(y_true, y_pred,
+    ...                   sensitive_features=sf_data)
+    0.19999...
+    >>> MetricFrame(metrics=recall_score,
     ...             y_true=y_true,
     ...             y_pred=y_pred,
-    ...             sensitive_features=group_membership_data).difference()
-    0.752525...
+    ...             sensitive_features=sf_data).difference()
+    0.19999...
 
 We use :func:`fairlearn.metrics.make_derived_metric` to manufacture a number
 of such functions which will be commonly used:
```

Comment:
```suggestion
We use :func:`fairlearn.metrics.make_derived_metric` to manufacture a number
of such functions which are commonly used. The table below displays the aggregations 
that are compatible with each base metric:
```

Reply:
Like the addition sentence, but have tweaked slightly, since I don't think 'compatible' is quite the right word.